### DISCUSSION TITLE
Why Ventoy has options for both MBR and GPT? and some weird question around it

### DISCUSSION THREAD

**Discussion by kaioken32** [ventoy/Ventoy#1917] [Q&A]
I have made ventoy with choosing mbr partition which is the default option for it. But I dont know whether it is legacy or UEFI. But I can seem to boot from both legacy based (NO CSM or uefi is present in BIOS) pc and UEFI based (NO CSM or legacy mode present in uefi) pc..... and install windows from it

Question is why am I not seeing this type of error? 
![Screenshot 2022-10-15 170914](https://user-images.githubusercontent.com/111365070/195983291-75e1f631-f3be-42d0-bb4b-a00e868d6faa.png)

**Comment by steve6375** [comment]
Are you legacy booting or UEFI-booting to the Ventoy USB drive?
The type of partitioning on the Ventoy USB disk is irrelevant.
You can legacy and UEFI-boot from MBR or GPT disks.
It is only Windows Setup that insists that if you legacy-boot you must use MBR partitions when installing Windows, or if you UEFI-boot then you must use GPT partitions when installing Windows.

**Reply by kaioken32** [reply]
Yep, I use both legacy based pc and uefi based, but I meant that how ventoy for example telling my uefi pc to install windows when my ventoy USB is of MBR parititon?

**Reply by steve6375** [reply]
The usb partition style does not matter.
You can uefi boot from an MBR usb Drive.
You can legacy boot from gpt usb drive.
Please read carefully my previous post!

**Comment by ruftas** [comment]
just wipe all partitions on disk.
